EventForwarding/EventForwarding_CZ_regulator.md

```diff
@@ -364,3 +364,51 @@ Ukázka zaslané události:
     "changeReason": "pre-heating-started"
 }
 ```
+
+### EventType physical-device-assigned
+Událost je odeslána při přiřazení fyzického zařízení k již existujícímu zařízení. Například, když je k regulátoru teploty přiřazena nová termostatická hlavice nebo teploměr.
+
+Dodatečné předávané parametry:
+
+| Parametr          | Typ    | Povinný | Popis                                                                                       |
+|:------------------|:-------|:--------|:--------------------------------------------------------------------------------------------|
+| assignedPhysicalDeviceId|	string|	ano|	Id přiřazeného fyzického zařízení. |
+| physicalDeviceType|	string (enum)|	ano|	Typ přiřazeného fyzického zařízení. Může nabývat hodnot - thermo-head a thermometer|
```

Comment:
thermo-head
thermostat
hygrometer-thermometer

Reply:
This is partner physical device type. Which contains only ThermoHead = 0,
    Thermometer = 1,